data-reconciliation-app/governance/nodes/cchost_config_enclave_js.json

```diff
@@ -8,7 +8,7 @@
     "node_to_node_interface": { "bind_address": "127.0.0.1:8081" },
     "rpc_interfaces": {
       "main_interface": {
-        "bind_address": "127.0.0.1:8080"
+        "bind_address": "0.0.0.0:8080"
       }
     }
   },
```

Comment:
Don't we need this to be configured in the same fashion as `cchost_config_virtual_js.json` ?
(using the IP we specify in `make` targets for docker...)

Reply:
Before my change - there was 2 config files. I've just copied the value that was in the file that was being used. This would have to be tested in Azure to know if it works